src/viewProviders/flinkArtifacts.test.ts

```diff
@@ -1,292 +1,114 @@
 import * as assert from "assert";
 import * as sinon from "sinon";
 import { CancellationToken, Progress, window } from "vscode";
-import { getStubbedCCloudResourceLoader } from "../../tests/stubs/resourceLoaders";
-import { createFlinkArtifact } from "../../tests/unit/testResources/flinkArtifact";
 import { TEST_CCLOUD_FLINK_COMPUTE_POOL } from "../../tests/unit/testResources/flinkComputePool";
-import { getTestExtensionContext } from "../../tests/unit/testUtils";
-import { ResponseError } from "../clients/flinkArtifacts";
 import * as errors from "../errors";
-import { CCloudResourceLoader } from "../loaders";
 import * as notifications from "../notifications";
-import { FlinkArtifactsViewProvider } from "./flinkArtifacts";
+import { FlinkArtifactsUDFsViewProvider } from "./flinkArtifacts";
 
-describe("FlinkArtifactsViewProvider", () => {
```

Comment:
Most of these tests moved to `src/viewProviders/multiViewDelegates/flinkArtifactsDelegate.test.ts`